src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -1066,4 +1066,22 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
     </xsd:complexType>
 
+    <xsd:complexType name="type-info" mixed="true">
+        <xsd:sequence>
+            <xsd:element name="aliases" minOccurs="0" maxOccurs="1">
```

Comment:
this does not make sense. There won't be any `aliases` element. The `alias` element must be a direct child of `<type-info>`. That's how prototyped nodes work in XML config files.